src/yunohost/dyndns.py

```diff
@@ -66,6 +66,38 @@ def __init__(self, line):
     r'.*/K(?P<domain>[^\s\+]+)\.\+157.+\.private$'
 )
 
+def _dyndns_provides(provider, domain):
+
+    logger.debug("Checking if %s is managed by %s ..." % (domain, provider))
+
+    try:
+        r = requests.get('https://%s/domains' % provider)
```

Comment:
We should add timeout here (we hadn't made this decision at the time you've made those commits).